analysis/src/topObjectSelection.cc

```diff
@@ -207,3 +240,95 @@ vector<TParticle> topObjectSelection::bjetSelection() {
   }
   return bjets;
 }
+
+
+// In uncertainty study we need to switch the kinematic variables of jets
+// The following variables are for this switch
+// In the topObjectSelection.cc these variables are used instead of Jet_pt, Jet_mass, and so on.
+// In default, these are same as the original ones, but when a user wants to study systematic uncertainty 
+// so that he/she needs to switch them to the evaluated ones, 
+// just touching them in anlalyser class will be okay, and this is for it.
+void topObjectSelection::GetJetMassPt(UInt_t nIdx, 
```

Comment:
Doesn't everyone need to do this? In which case, why are we doing it in top objects and not a level above?

Reply:
You mean, it would be better if this function (and the stuffs for initialization in the constructor) is in mother class, that is, nanoBase?